example/client/agent/main.go

```diff
@@ -0,0 +1,192 @@
+//
```

Comment:
this file is about the same as `example/client/main.go`, user may not notice what is the differences between... 🤔🤔

Reply:
I don't think there is any problem even if they are similar.
I don't think it's a problem to be similar. Rather, I feel that it is better not to be common and complicated. So I'm trying to explain it carefully in the comments.